### ISSUE TITLE
Pretrained SavedModel Issues

### ISSUE THREAD

**Issue by cgarciae** [google-research/simclr#2]
First of all thanks a lot for releasing this! Its a huge contribution for the whole industry.

I've explored the pretrained model you released and got it working, I have some small issues that could improve its usage:

1. Use the the `serving_default` signature which I think its the standard.
2. Some documentation around the inputs and outputs of the model. I got that `inputs` are `(batch_size, 224, 224, 3)`, the `default` output tensor is the embeddings, and `*logits*` output tensor is the output when training, don't know about the other outputs.

**Comment by chentingpc** [comment]
Thanks for the feedback! Here is a list of signatures for hub module:

```
{'block_group1':` <tf.Tensor 'module_apply_default/base_model/block_group1:0' shape=(1, 56, 56, 256) dtype=float32>,
 'logits_sup': <tf.Tensor 'module_apply_default/head_supervised/linear_layer/linear_layer_out:0' shape=(1, 1000) dtype=float32>,
 'default': <tf.Tensor 'module_apply_default/base_model/Squeeze:0' shape=(1, 2048) dtype=float32>,
 'initial_max_pool': <tf.Tensor 'module_apply_default/base_model/initial_max_pool:0' shape=(1, 56, 56, 64) dtype=float32>,
 'initial_conv': <tf.Tensor 'module_apply_default/base_model/initial_conv:0' shape=(1, 112, 112, 64) dtype=float32>,
 'block_group4': <tf.Tensor 'module_apply_default/base_model/block_group4:0' shape=(1, 7, 7, 2048) dtype=float32>,
 'final_avg_pool': <tf.Tensor 'module_apply_default/base_model/final_avg_pool:0' shape=(1, 1, 1, 2048) dtype=float32>,
 'block_group2': <tf.Tensor 'module_apply_default/base_model/block_group2:0' shape=(1, 28, 28, 512) dtype=float32>,
 'block_group3': <tf.Tensor 'module_apply_default/base_model/block_group3:0' shape=(1, 14, 14, 1024) dtype=float32>}
```
The default and final_avg_pool are both the same, and they're the representation output of the base network; logits_sup is the supervised classification logits for ImageNet 1000 categories. Others are middle layers of ResNet, see resnet.py for the specifics.

Let me know if you have any further questions.

**Comment by cgarciae** [comment]
Thanks for clarifying!
